### ISSUE TITLE
Give the client an option to receive an email to sign

### ISSUE THREAD

**Issue by sineadvaughan** [SuffolkLITLab/docassemble-NoticeOfLimitedAppearance#11]

**Comment by sineadvaughan** [comment]
Provide an option for the Client to receive a link by email to sign the form.

**Comment by plocket** [comment]
Is this MVP?

**Comment by CaroRob** [comment]
yes - I think @miabonardi  is on it though.  Attorney cannot file this form if client has not signed it. In this climate best way for client to sign is via email or text, but I also think this issue is actually working and resolved - @miabonardi ?

**Comment by miabonardi** [comment]
@plocket @CaroRob This issue is not resolved. Currently, the user (attorney) can only send the form to the client via text. If the client is unavailable, the attorney can still fill out the form, but the form will not be sent to the court.
